initfiles/bash/etc/init.d/hpcc_common.in

```diff
@@ -505,6 +505,8 @@ startCmd() {
       SENTINEL_CHECK=0
     fi
 
+    # sleep is necessary to insure the eval $startcmd actually kicked off
+    sleep 1
```

Comment:
I'm always suspicious of sleep's like this.
eval returns the exit status of the command, but I guess start-stop-daemon returns before the service process actually starts, is that why the sleep was added?

What if the node is busy and takes > 1 sec?
Why is this new, is it a very small window (normally) that was observed somewhere?